client/core/core.go

```diff
@@ -1931,18 +1952,37 @@ func (c *Core) initialize() {
 		wg.Add(1)
 		go func(acct *db.AccountInfo) {
 			defer wg.Done()
+			host := addrHost(acct.Host)
 			dc, err := c.connectDEX(acct)
 			if err != nil {
 				log.Errorf("error connecting to DEX %s: %v", acct.Host, err)
 				return
 			}
 			log.Debugf("connectDEX for %s completed, checking account...", acct.Host)
+			if acct.Disabled {
+				return
+			}
+			if !acct.Disabled && c.conns[host] != nil && c.conns[host].acct != nil {
```

Comment:
I've removed the redundant `if acct.Disabled {` call.

The situation of having multiple enabled client accounts arose as I was testing the disable capability. Sometimes the disable call would not occur during initializeDEXConnections() when it was supposed to which would leave multiple enabled client accounts in the client db. 

The nature of processing the accounts in initialize() is to put the account into the `c.conns[host] = dc` map. Multiple enabled client accounts would continue to remain in the client db. Processing within initializeDEXConnections() would not solve this as it only iterates over c.conns by host and therefore would only manage/disable whatever client account happened to be the last one placed into c.conns.

Perhaps this situation will not occur in the real world, but it did happen to me while testing. Due to the fact that if it occurs it may lead to erratic behavior such as the user being prompted to re-register because the client account that happened to make it into c.conns[host] is no longer found on the dex, while another client account that would successfully authenticate against the dex is randomly hidden.

I considered 2 other options.

1. Execute `c.authDEX(dc)` during initialize(). I'm not a fan of this as it seems wrong to authenticate before the user has requested to authenticate.

2. Query for client db accounts by host in initializeDEXConnections() to address multiple enabled client accounts. I preferred to address it in initialize() as it is already iterating all of the client db accounts.

I do agree that it would be nice to have set criteria determining which client account to disable rather than randomly disabling the first ones that are recognized as duplicates. I'm thinking this is not easily addressed without UI work that allows the user to choose which account they want to use or leave enabled.